Cargo.toml

```diff
@@ -92,3 +93,9 @@ anvil_zksync_api_server = { path = "crates/api_server" }
 anvil_zksync_config = { path = "crates/config" }
 anvil_zksync_core = { path = "crates/core" }
 anvil_zksync_types = { path = "crates/types" }
+zksync_error = { path = "crates/zksync_error" }
+
+#########################
+# Build dependencies    #
+#########################
+zksync-error-codegen = { git = "https://github.com/matter-labs/zksync-error", branch = "main" }
```

Comment:
We should reference a commit hash here instead unless we want to make an initial release of zksync-error?

Reply:
Maybe rather a tag? 
There are two locations in where it is important: here and in [build.rs](https://github.com/matter-labs/anvil-zksync/pull/583/files#diff-04f920aafdbfbf984dddc44731bd9477f37c0b7755e7113435eb24a2680c8009R16).
Something like `stable`.
This is a build dependency to rebuild the `zksync-error` crate, which is included. It is not triggered on build unless `anvil.json` is changed, therefore even if repository `zksync-error` breaks compatibility, `anvil-zksync` is still built without issues.

Reply:
We need to start publishing it on `crates.io` (thankfully we have an automation for that now), but until then I'd prefer to depend on commit hash. Everything else can be a PITA to manage due to `Cargo.lock` (you can update the dep in `Cargo.toml`, but it won't be updated in `Cargo.lock` until you do `cargo update`).

Reply:
+1 to commit hash